### PR TITLE
FSET-1616 Pass skills as query parameter list

### PR DESCRIPTION
**Pull request by benjstephenson**

### CODE REVIEW

File: app/controllers/AssessorController.scala [hunk 1]
```diff
@@ -73,8 +73,16 @@ trait AssessorController extends BaseController {
...
-    skills: List[SkillType]): Action[AnyContent] = Action.async { implicit request =>
+    skills: Seq[SkillType]
+  ): Action[AnyContent] = Action.async { implicit request =>
     assessorService.findAvailabilitiesForLocationAndDate(locationName, date, skills).map { a => Ok(Json.toJson(a)) }
   }
 
+  def findAllocations(assessorId: String): Action[AnyContent] = Action.async { implicit request =>
+    assessorService.findAllocations(assessorId).map(allocations => Ok(Json.toJson(allocations)))
+		  .recover {
+				case e: Exception => InternalServerError(e.getMessage)
```
Comment (kjharkin) [inline app/controllers/AssessorController.scala]: what's the thinking behind returning a 500 explicitly?

File: app/controllers/package.scala [hunk 2]
```diff
@@ -32,19 +33,19 @@ package object controllers {
...
 
-    /*implicit object skillTypeQueryBinder extends QueryStringBindable.Parsing[SkillType](
-       parse = SkillType.withName(_),
-       serialize = _.toString,
-       error = (m: String, e: Exception) => "Can't parse %s as SchemeType : %s".format(m, e.getMessage)
-    )*/
-    private def enumQueryBinder[E <: Enumeration](enum: E) = {
+    private def enumBinder[E <: Enumeration](enum: E) = {(
       new QueryStringBindable.Parsing[E#Value](
         parse = enum.withName(_),
```
Comment (kjharkin) [inline app/controllers/package.scala]: (style?) ```parse = enum.WithName``` is all you need

File: app/controllers/package.scala [hunk 3]
```diff
@@ -32,19 +33,19 @@ package object controllers {
...
         serialize = _.toString,
-        error = (m: String, e: Exception) => "Can't parse %s as SchemeType : %s".format(m, e.getMessage)
+        error = (m: String, e: Exception) => "Can't parse %s as %s : %s".format(m, enum.getClass.getSimpleName, e.getMessage)
+      ),
+      new mvc.PathBindable.Parsing[E#Value](
+        parse = enum.withName(_),
```
Comment (kjharkin) [inline app/controllers/package.scala]: ditto

**Comment by kjharkin** [comment]
Good stuff - see (very minor comments) and I'll refactor.

File: app/controllers/AssessorController.scala (hunk 1 above)
Comment (benjstephenson) [inline app/controllers/AssessorController.scala]: Not a single clue!  I'll remove it.

**Pull request merged**